### DISCUSSION TITLE
Um help para Aplicação com JS.

### DISCUSSION THREAD

**Discussion by marcusvcribeiro** [frontendbr/forum#1000] [Perguntas e Respostas]
Fala Galera,

Estou com um problema aqui no trabalho, pediram para fazer uma área administrativa que irá pegar fotos por #(hashtag), só que não faço a menor ideia de como fazer isso.
Algumas buscas no google não me ajudaram tb, então decidi vir aqui ver o que vcs usariam para fazer algo assim.

**Comment by felipefialho** [comment]
Especifique melhor sua dúvida. 

- Que hashtags são essas?
- São associadas ao Instagram ou outra rede social?

**Comment by woliveiras** [comment]
Fala, @marcusvcribeiro!

Cara,

- Área administrativa em qual plataforma? Ou você vai criar do zero?
- Existe API que devolva essas fotos e você vai filtrar pela tag?

**Comment by leandroruel** [comment]
acho que é pra consumir a API do Instagram, e pegar as fotos pelas hashtags dos usuários. acho.

**Comment by marcelogbrito** [comment]
Se for isso:
1) consiga uma chave no instagram para consumir a api https://www.instagram.com/developer/
2) consuma esse endpoint (para busca por tags) com a aplicação na linguagem da sua preferencia https://instagram.com/developer/endpoints/tags/

**Comment by bernardodiasc** [comment]
A API do Instagram ta sendo descontinuada, daqui alguns meses será preciso utilizar a Graph API do Facebook. Escrevi um post sobre isso aqui http://bernardodiasdacruz.com/2018/03/05/fetching-from-instagram-using-facebook-graph-api
